lisa/transformers/kernel_installer.py

```diff
@@ -135,7 +135,10 @@ def _internal_run(self) -> Dict[str, Any]:
             posix = cast(Posix, node.os)
             posix.replace_boot_kernel(installed_kernel_version)
 
-        if installer.runbook.source == "linux-image-azure-fde":
+        if (
```

Comment:
It can be merged with above `if`. So the logic is clearer to handle different behavior of fde/non-fde.

Reply:
Sure, Raised PR: https://github.com/microsoft/lisa/pull/2154